src/App.java

```diff
@@ -13,17 +13,76 @@ public void start() {
         printGreeting(userName);
         printInstructions();
         doMadlib();
-        System.out.println("hi");
+      
+
+        
     }
-    
+    public String getUserName(){
+  Scanner scanner = new Scanner(System.in);
+  System.out.print("Enter your name: ");
+   return scanner.nextLine();
+    }
+   
+
+
+
+
+   public void printGreeting(String userName){
+   System.out.println("Hello," + userName + "!");
+
+   }
+
+    public void printInstructions(){
+ System.out.println("To play this game you have to fill in the boxes with words that corresponds with what is asked.");
```

Comment:
Your indentation here, and in several other places, is not correct. Most coding tools, including VS Code, will help you format your code, but it's a good idea to try to keep it mostly formatted yourself as you write, it makes it easier for you to read your own work as well as for others to read it.